deployments/helm-chart/README.md

```diff
@@ -215,6 +243,10 @@ Parameter | Description | Default
 `controller.reportIngressStatus.annotations` | The annotations of the leader election configmap. | {}
 `controller.pod.annotations` | The annotations of the Ingress Controller pod. | {}
 `controller.appprotect.enable` | Enables the App Protect module in the Ingress Controller. | false
+`controller.appprotectdos.enable` | Enables the App Protect Dos module in the Ingress Controller. | false
+`controller.appprotectdos.debug` | Enables debug logs for App Protect Dos | false
+`controller.appprotectdos.arbitrator.name` | The name of the App Protect Dos Arbitrator deployment. | Autogenerated
```

Comment:
what does `Autogenerated` mean? if it's autogenerated why do we need a parameter?

Reply:
I took it from nginx-ingress name:

controller:
  ## The name of the Ingress controller daemonset or deployment.
  ## Autogenerated if not set or set to "".
  # name: nginx-ingress

this parameter change the autogenerated arbitrator pod name to the name that given.
arbitrator:
  name: appprotect-dos-arb
Pod name will be:
from:
my-release-nginx-ingress-appprotect-dos-arb-86f5c89894-rsqxh
to:
appprotect-dos-arb-598f6647b8-gxzzh

Reply:
Remove the name option and autogenerated will be only option

Reply:
Done